cmd/syft/cli/commands/update.go

```diff
@@ -1,31 +1,64 @@
-package version
+package commands
 
 import (
 	"fmt"
 	"io"
 	"net/http"
 	"strings"
 
+	"github.com/wagoodman/go-partybus"
+
+	"github.com/anchore/clio"
 	hashiVersion "github.com/anchore/go-version"
-	"github.com/anchore/syft/internal"
+	"github.com/anchore/syft/cmd/syft/internal"
+	"github.com/anchore/syft/internal/bus"
+	"github.com/anchore/syft/internal/log"
+	"github.com/anchore/syft/syft/event"
 )
 
 var latestAppVersionURL = struct {
 	host string
 	path string
 }{
 	host: "https://toolbox-data.anchore.io",
-	path: fmt.Sprintf("/%s/releases/latest/VERSION", internal.ApplicationName),
+	path: fmt.Sprintf("/%s/releases/latest/VERSION", internal.Syft),
```

Comment:
this could probably be hardcoded now, then we can drop the very-tempting-to-reference `internal.Syft` variable entirely